lint.swift

```diff
@@ -0,0 +1,547 @@
+
+    // MARK: ChangelogEntryTrailingWhitespaces
+    try Lint.checkFileContents(
+        checkInfo: "ChangelogEntryTrailingWhitespaces: The summary line of a Changelog entry should end with two whitespaces.",
+        regex: #"\n([-–] (?!None\.).*[^ ])( {0,1}| {3,})\n"#,
+        matchingExamples: ["\n- Fixed a bug.\n  Issue:", "\n- Added a new option. (see [Link](#)) \nPR:"],
+        nonMatchingExamples: ["\n- Fixed a bug.  \n  Issue:", "\n- Added a new option. (see [Link](#))  \nPR:"],
+        includeFilters: [changelogFile],
+        autoCorrectReplacement: "\n$1  \n",
+        autoCorrectExamples: [
+            ["before": "\n- Fixed a bug.\n  Issue:", "after": "\n- Fixed a bug.  \n  Issue:"],
+            ["before": "\n- Fixed a bug. \n  Issue:", "after": "\n- Fixed a bug.  \n  Issue:"],
+            ["before": "\n- Fixed a bug.   \n  Issue:", "after": "\n- Fixed a bug.  \n  Issue:"],
+            ["before": "\n- Fixed a bug !\n  Issue:", "after": "\n- Fixed a bug !  \n  Issue:"],
+            ["before": "\n- Fixed a bug ! \n  Issue:", "after": "\n- Fixed a bug !  \n  Issue:"],
+            ["before": "\n- Fixed a bug !  \n  Issue:", "after": "\n- Fixed a bug !  \n  Issue:"],
+        ]
+    )
+
+    // MARK: ChangelogEntryLeadingWhitespaces
+    try Lint.checkFileContents(
+        checkInfo: "ChangelogEntryLeadingWhitespaces: The links line of a Changelog entry should start with two whitespaces.",
+        regex: #"\n( {0,1}| {3,})(Tasks?:|Issues?:|PRs?:|Authors?:)"#,
+        matchingExamples: ["\n- Fixed a bug.\nIssue: [Link](#)", "\n- Fixed a bug. \nIssue: [Link](#)", "\n- Fixed a bug.    \nIssue: [Link](#)"],
+        nonMatchingExamples: ["- Fixed a bug.\n  Issue: [Link](#)"],
+        includeFilters: [changelogFile],
+        autoCorrectReplacement: "\n  $2",
+        autoCorrectExamples: [
+            ["before": "\n- Fixed a bug.\nIssue: [Link](#)", "after": "\n- Fixed a bug.\n  Issue: [Link](#)"],
+            ["before": "\n- Fixed a bug.\n Issue: [Link](#)", "after": "\n- Fixed a bug.\n  Issue: [Link](#)"],
+            ["before": "\n- Fixed a bug.\n    Issue: [Link](#)", "after": "\n- Fixed a bug.\n  Issue: [Link](#)"],
+        ]
+    )
+
+    // MARK: ClosureParamsParantheses
+    try Lint.checkFileContents(
+        checkInfo: "ClosureParamsParantheses: Don't use parantheses around non-typed parameters in a closure.",
+        regex: #"\{\s*\(((?!self)[^):]+)\)\s*in"#,
+        matchingExamples: ["run { (a) in", "run { (a, b) in", "run { (a, b, c) in"],
+        nonMatchingExamples: ["run { (a: Int) in", "run { (a: Int, b: Int) in", "run { (a, b) -> String in"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: #"{ $1 in"#,
+        autoCorrectExamples: [
+            ["before": "run { (a) in", "after": "run { a in"],
+            ["before": "run { (a, b) in", "after": "run { a, b in"],
+            ["before": "run { (a, b, c) in", "after": "run { a, b, c in"]
```

Comment:
Codacy found an issue: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)